Tasks/VSMobileCenterTest/Tests/L0.ts

```diff
@@ -119,21 +119,21 @@ describe('VSMobileCenterTest L0 Suite', function () {
     it('Negative path: with username and password, should always logout even test prepare failed', (done: MochaDone) => {
```

Comment:
I think this description needs to be updated.  I understand we don't login/logout for prepare step anymore.